### PR TITLE
ADR001_Tech_Choice

### PR DESCRIPTION
**Pull request by susmithakunisetty**
First ADR for the Technology Choices

### CODE REVIEW

File: docs/decisions/ADR001_TechnologyDecisions.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Pros and Cons of the Options
+
+### Opt for Open Source Technologies
+
+* Good, because opting for Open Source software enables flexibility to integrate with other technologies
+* Good, because it is cost effective, stable and performant with large development community support.
+* Good, because No Vendor locking
+* Good, because it offers cost savings
+* Bad, because of the security treats, however adopting a secured underlying framework.
```
Comment (mgasca) [inline docs/decisions/ADR001_TechnologyDecisions.md:36]: Threats is misspelled.

I'm not sure I understand this point.  All software is at risk of security vulnerabilities.  Also it's not clear what is meant by "adopting a secured underlying framework"

File: docs/decisions/ADR001_TechnologyDecisions.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+* Bad, because of the Technical Debt issue.
```
Comment (mgasca) [inline docs/decisions/ADR001_TechnologyDecisions.md:37]: What technical debt issue?

File: docs/decisions/ADR001_TechnologyDecisions.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
```
Comment (mgasca) [inline docs/decisions/ADR001_TechnologyDecisions.md:38]: Nee a section for pros/cons of packaged software
Pros:
Higher service level for support with paid contracts when issues arise
Cons:
Higher cost
Higher cost due to licensing and support

File: docs/decisions/ADR001_TechnologyDecisions.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
```
Comment (mgasca) [inline docs/decisions/ADR001_TechnologyDecisions.md:39]: We should add a consequences section

```suggestion
## Consequences
As open source quality, evolution and support relies on the level of engagement and vibrancy of the community that surrounds them, we must ensure to take this into account with every choice we make.
```

**Review by mgasca — COMMENTED** [review]
The Flow jpgs don't belong in this PR.  They should be submitted separately.  Furthermore, if we agree to use draw.io then we should commit the xml files along with any images so that the diagrams can be recreated/edited.

**Comment by mgasca** [comment]
Associating this with issue #2

File: docs/decisions/ADR001_TechnologyDecisions.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
```
Comment (mgasca) [inline docs/decisions/ADR001_TechnologyDecisions.md:1]: Based on convention in MADR format/template/conventions, the filename should start with `0001-` not `ADR001_`

File: docs/decisions/ADR001_TechnologyDecisions.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+## Decision Drivers
+
+* Building a cost effective application
+* Extendable and Flexibility to integrate with other technologies
+* Hige development community
+
+## Considered Options
+
+* Opt for Open Source Technologies
```
[the remark this answers is not in the document]
Comment (susmithakunisetty) [reply docs/decisions/ADR001_TechnologyDecisions.md]: Incorporated the review comments @mgasca

**Review by mgasca — APPROVED** [review]
Thanks @susmithakunisetty,  PR approved.  I'm merging.

**Pull request merged**